### DISCUSSION TITLE
How to write a proper machine.json file to run dpgen on my own computer?

### DISCUSSION THREAD

**Discussion by Tsuki95** [deepmodeling/dpgen#592] [Q&A]
Hi, I am a beginner of Dpgen. I have tried many different example machine files and have changed them to suit my computer, but it always turned in different errors. Has anyone used Dpgen in your own computer environment and could probably share an example machine file with me? Thanks a lot!!!

**Comment by McAlexDudu** [comment]
Do you solve the problem now?
